src/frontend/src/pages/CreateProjectPage/CreateProjectFormView.tsx

```diff
@@ -122,13 +122,10 @@ const CreateProjectFormView: React.FC<CreateProjectFormViewProps> = ({ allowSubm
             </FormControl>
           </Grid>
         </Grid>
-        <Box display="flex" gap={2} sx={{ mt: 2 }}>
-          <SubmitButton variant="contained" color="primary" type="submit" disabled={!allowSubmit}>
-            Create
-          </SubmitButton>
-          <Button variant="outlined" color="secondary" onClick={onCancel}>
-            Cancel
-          </Button>
```

Comment:
Make sure you include the NERFailButton with the cancel button as well, that's probably what's causing the tests to fail

Reply:
Yep I agree with Reid, on the old lines 129-131, there is code for the 'cancel' button. You still want this code but need to replace certain parts of it with 'NERFailButton'.